src/app/pages/home-page/components/faqs-section/faqs-section.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Faq} from '../../../../models/faq';
+
+@Component({
+  selector: 'app-faqs-section',
+  templateUrl: './faqs-section.component.html',
+  styleUrls: ['./faqs-section.component.css']
+})
+export class FaqsSectionComponent implements OnInit {
+
+  @Input() title: string;
+  @Input() description: string;
```

Comment:
You will only use this component once with a fixed title and description. So you don't need to add them as an input property. Input is used when you will use the component multiple times throughout your project and you need some parameters to change every time.